### ISSUE TITLE
Chatlog isn't sent

### ISSUE THREAD

**Issue by DeinFreund** [ZeroK-RTS/Zero-K-Infrastructure#2139]
I don't always get shown messages since my last logout when I login into Chobby. Maybe it has to do with improper logout/connection loss.

**Comment by Licho1** [comment]
yes if you disconnect logout might come later and you gt less data

**Comment by DeinFreund** [comment]
Im talking about missing an entire night's worth of messages. I'll still have to see how to reliably reproduce it though.

**Comment by DeinFreund** [comment]
I can reproduce it like this:
Press logout in chobby, write something in discord, press login, see the message sent in chatlog.
Press logout in chobby, close chobby. Write something in discord. Start lobby and login, don't see the message sent in chatlog.

**Comment by DeinFreund** [comment]
The server always throws a 
```
ZkData.TcpTransport error sending command: Unable to write data to the transport connection: An established connection was aborted by the software in your host machine.
```
When I do the latter.

**Comment by DeinFreund** [comment]
I don't get any chatlogs anymore.
